src/components/youthNet/BottomDrawer.tsx

```
@@ -62,32 +65,36 @@ const BottomDrawer: React.FC<BottomDrawerProps> = ({
+              textAlign: 'left',
+              marginBottom: '12px',
+              '&:hover': {
+                backgroundColor: 'transparent',
+                border: 'none',
+              },
+            }}
+            onClick={button.onClick}
+          >
+            {button?.label}
+          </Button>
+          <Divider />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)